vars/buildPlugin.groovy

```diff
@@ -48,22 +48,21 @@ def call(Map addonParams = [:])
 			'groovy',
 			'focal',
 			'bionic',
-			'xenial'
 		],
 		'nightly': [
+			'hirsute',
 			'groovy',
 			'focal',
 			'bionic'
 		]
 	]
 	def PPAS_VALID = [
 		'nightly': 'ppa:team-xbmc/xbmc-nightly',
-		'unstable': 'ppa:team-xbmc/unstable',
```

Comment:
Yes makes sense to remove unstable, never really used and think there not so up to date related to depends as on nightly and stable :+1: